kedro/io/core.py

```diff
@@ -66,7 +66,11 @@ class VersionNotFoundError(DataSetError):
     pass
 
 
-class AbstractDataSet(abc.ABC):
+_DI = TypeVar("_DI")
+_DO = TypeVar("_DO")
```

Comment:
What do `_DI` and `_DO` stand for?

Reply:
They refer to `DataInput` (save) and `DataOutput` (load), but I'd be happy with changing it to something else. Generics are usually typed with one- or two-letter acronyms, and these two made the most sense to me.

Reply:
I like it 👍

Reply:
Okay that makes sense! I'd just leave a comment that's what they refer to for future reference.